assets/booking_index.js

```diff
@@ -58,8 +59,10 @@ document.addEventListener('DOMContentLoaded', function() {
 
     function setBookingId(info){
         bookingId = info.event.id
+        document.getElementById('bookingEdition').setAttribute('href', bookingId + '/edit')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/LeonardDeVanGogh/VosgesActivies/pullRequest?prid=7268173)
```suggestion
        document.getElementById("bookingEdition").setAttribute('href', bookingId + '/edit')
```